src/main/kotlin/fi/metatavu/timebank/api/impl/UsersApi.kt

```diff
@@ -0,0 +1,37 @@
+package fi.metatavu.timebank.api.impl
+
+import fi.metatavu.timebank.api.controllers.UsersController
+import fi.metatavu.timebank.spec.UsersApi
+import java.util.*
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+import javax.ws.rs.core.Response
+
+@RequestScoped
+class UsersApi: UsersApi, AbstractApi() {
```

Comment:
It has been a convention in this project that API implementation has class level documentation, please include it. Even if `Implementation for Users API` does seem self documenting.